app/controllers/employers/employer_attestations_controller.rb

```diff
@@ -25,6 +25,14 @@ def verify_attestation
     @document = attestation.employer_attestation_documents.find(params[:employer_attestation_id])
   end
 
+  def revert_attestation
+    attestation = @employer_profile.employer_attestation
+    @document = attestation.employer_attestation_documents.find(params[:employer_attestation_id])
+    @document.submit! if @document.may_submit?
+    attestation.submit! if attestation.may_submit?
+    redirect_to employer_invoice_exchanges_hbx_profiles_path
+  end
+
```

Comment:
As, @dominic and @raghuramg, suggested there are some additions to the functionality. I`ll take care of your suggestions for both existing and additional functionality specs, while implementing the new functionality. Thank you @brianweiner